qiskit/circuit/library/standard_gates/global_phase.py

```diff
@@ -0,0 +1,75 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Global Phase Gate"""
+
+from typing import Optional  # , Union
+import numpy
+
+# from qiskit.qasm import pi
+
+# from qiskit.circuit.controlledgate import ControlledGate
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.quantumregister import QuantumRegister
+
+# from qiskit.circuit._utils import _compute_control_matrix
+from qiskit.circuit.quantumcircuit import QuantumCircuit
+from qiskit.circuit.parameterexpression import ParameterValueType
+
+# from qiskit.circuit.exceptions import CircuitError
+
+
+class GlobalPhaseGate(Gate):
+    r"""The global phase gate (:math:`e^{i\theta}`).
+
+    Can be applied to a :class:`~qiskit.circuit.QuantumCircuit`
+
+    **Mathamatical Representation:**
+
+
+    .. math::
+        \text{GlobalPhaseGate}\ =
+            \begin{pmatrix}
+                e^{i\theta}
+            \end{pmatrix}
+
+    **Circuit symbol:**
+
+    .. parsed-literal::
```

Comment:
When this class is actually added to the documentation build (once `GlobalPhaseGate` is in the correct place in `qiskit/circuit/library/__init__.py`'s docstring), I think this will likely cause a documentation build failure because there's no block after the `parsed-literal`.  In this case, I think it'll be best just to remove the "Circuit symbol" heading and the `parsed-literal` directive completely, since there's nothing (currently) to show.